packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -3415,6 +3415,15 @@ export class API extends EventEmitter {
     });
   }
 
+  sardineGetSupportedTokens(data): Promise<any> {
```

Comment:
A BWC bump will not be necessary. By not using getServerWithAuth we can access through our BWS URL directly (.../v1/service/sardine/getSupportedTokens). So you can remove this function.